src/Components/LinkItem.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
Was this code added in this pull request, or was it added previously?

Reply:
I think it's already in code, as I just made tiny change at Link Button

Reply:
This whole file is new, I don't see it in the default branch

Reply:
I can see that you are making all the PR from your main branch. 
I have a suggestion for the problem of all the PR coming under one PR. Make a new branch from the main branch for every PR you make for any new issue.
Hopefully, this will help you. 😃